Cargo.toml

```diff
@@ -70,7 +70,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 snafu = { version = "0.7", features = ["backtraces"] }
 sqlparser = "0.30"
-tempfile = "3"
+tempfile = "3.4.0"
```

Comment:
Prefer the version to be `3.4` based on our code style guide.

Reply:
Got it, will adjust